SQL/database_changelog.txt

```diff
@@ -1,3 +1,23 @@
+28 June 2017, by oranges
+Added schema_revision to store the current db revision, why start at 3.0?
+
+because:
+15:09 <+MrStonedOne> 1.0 was erro, 2.0 was when i removed erro_, 3.0 was when jordie made all the strings that hold numbers numbers
+
+CREATE TABLE `schema_revision` (
+`major` TINYINT(3) UNSIGNED NOT NULL ,
+`minor` TINYINT(3) UNSIGNED NOT NULL ,
+`date` DATETIME DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP NOT NULL,
+PRIMARY KEY ( `major`,`minor` ),
+) ENGINE = INNODB;
+
+INSERT INTO `schema_revision`
+(`major`, `minor`, `date`) VALUES (3, 0, SELECT CURRENT_TIMESTAMP);
```

Comment:
#1064 - You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'SELECT CURRENT_TIMESTAMP)' at line 2